modules/oadp-csi-snapshot-timeouts.adoc

```diff
@@ -13,6 +13,13 @@ Use the `CSISnapshotTimeout`  for the following scenarios:
 * With the CSI plugin.
 * For very large storage volumes that may take longer than 10 minutes to snapshot. Adjust this timeout if timeouts are found in the logs.
 
+[IMPORTANT]
+.Backup configuration "csiSnapshotTimeout" not functioning as expected
+====
+In OADP 1.2.0 and 1.2.1, the backup configuration setting `csiSnapshotTimeout` is not functioning as expected. When attempting to modify this parameter to a custom value, the backup process still utilizes the default timeout value instead of the specified one. This behavior persists regardless of the value assigned to `csiSnapshotTimeout`.
+This issue has been resolved in OADP 1.2.2. 
```

Comment:
```suggestion
In OADP 1.2.0 and 1.2.1, the backup configuration setting `csiSnapshotTimeout` is not functioning as expected. When attempting to modify this parameter to a custom value, the backup process still utilizes the default timeout value instead of the specified one. This behavior persists regardless of the value assigned to `csiSnapshotTimeout`. This issue has been resolved in OADP 1.2.2. 
```